### PR TITLE
fix: [OPA-666]: we need HPE to show on any enterprise license

### PR DESCRIPTION
**Pull request by kylekstewart**
### Summary

After discussion we need to display the HPE when it's any kind of Enterprise license. HPE service will take care on limiting functionality.

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Review by stuart-mcdonnell — APPROVED** [review]

**Comment by stuart-mcdonnell** [comment]
retrigger coverage

**Comment by stuart-mcdonnell** [comment]
trigger cypress

**Comment by stuart-mcdonnell** [comment]
trigger cypress

**Comment by stuart-mcdonnell** [comment]
trigger cypress

**Comment by stuart-mcdonnell** [comment]
retrigger coverage

**Comment by stuart-mcdonnell** [comment]
retrigger coverage

**Review by stuart-mcdonnell — APPROVED** [review]

**Review by crystal-fong — APPROVED** [review]

File: src/modules/10-common/hooks/useModuleLicenses.tsx [hunk 1]
```diff
@@ -12,9 +12,7 @@ import { useModuleInfo } from './useModuleInfo'
 export function useAnyEnterpriseLicense(): boolean {
   const { licenseInformation } = useLicenseStore()
 
-  const anyEntitlement = Object.values(licenseInformation).find(
-    license => license?.edition === Editions.ENTERPRISE && license?.status === 'ACTIVE'
-  )
+  const anyEntitlement = Object.values(licenseInformation).find(license => license?.edition === Editions.ENTERPRISE)
```
Comment (knagurski) [inline src/modules/10-common/hooks/useModuleLicenses.tsx:15]: You can simplify this if you swap from a `find` to a `some` and just return it.
```suggestion
  return Object.values(licenseInformation).some(license => license?.edition === Editions.ENTERPRISE)
```

File: src/modules/10-common/hooks/useModuleLicenses.tsx [hunk 2]
```diff
@@ -26,8 +24,7 @@ export function useCurrentEnterpriseLicense(): boolean {
     ? licenseInformation?.[module?.toUpperCase()]
     : { edition: 'FREE', status: 'EXPIRED' } // default so we don't display HPE if theres an error getting license/module info
 
-  const currentModuleEntitlement =
-    moduleEntitlement?.edition === Editions.ENTERPRISE && moduleEntitlement.status === 'ACTIVE'
+  const currentModuleEntitlement = moduleEntitlement?.edition === Editions.ENTERPRISE
```
Comment (knagurski) [inline src/modules/10-common/hooks/useModuleLicenses.tsx:27]: You could just return this instead of storing it to a variable first.

**Review by knagurski — APPROVED** [review]
Couple of suggestions, but no blockers. Nice simple PR 👍🏻

**Pull request merged**